packages/types/src/node.ts

```diff
@@ -28,6 +28,15 @@ export namespace Node {
     type: EventName;
   };
 
+  // This is used instead of the ethers `Transaction` because that type
+  // requires the nonce and chain ID to be specified, when sometimes those
+  // arguments are not known at the time of creating a transaction.
+  export type MinimalTransaction = {
```

Comment:
Aren't all properties optional on the ethers transaction?

Reply:
Apparently not.